concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1002,167 +1016,176 @@ jobs:
+            image: docker:///governmentpaas/cf-cli
+            inputs:
+              - name: paas-cf
+              - name: config
+              - name: bosh-CA
+            run:
+              path: sh
+              args:
+                - -e
+                - -c
+                - |
+                  ./paas-cf/concourse/scripts/import_bosh_ca.sh
 
-              . ./config/config.sh
-              echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
-              echo | cf create-org testers
-              cf create-space healthcheck -o testers
-              cf target -o testers -s healthcheck
+                  . ./config/config.sh
+                  echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+                  echo | cf create-org testers
+                  cf create-space healthcheck -o testers
+                  cf target -o testers -s healthcheck
 
-              cd paas-cf/tests/example-apps/healthcheck-app
+                  cd paas-cf/tests/example-apps/healthcheck-app
 
-              cf push
+                  cf push
 
-    - task: deploy-graphite-nozzle
-      config:
-        platform: linux
-        inputs:
-          - name: paas-cf
-          - name: graphite-nozzle
-          - name: config
-          - name: bosh-CA
-        params:
-        image: docker:///governmentpaas/cf-cli
-        run:
-          path: sh
-          args:
-            - -e
-            - -c
-            - |
-              ./paas-cf/concourse/scripts/import_bosh_ca.sh
-
-              . ./config/config.sh
-              echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
-              echo | cf create-org admin
-              cf create-space admin -o admin
-              cf target -o admin -s admin
-
-              cat <<EOF > graphite-nozzle.json
-              [
-                  {"protocol":"udp","destination":"${GRAPHITE_SERVER}","ports":"8125"}
-              ]
-              EOF
-              cf create-security-group graphite-nozzle graphite-nozzle.json
-              cf bind-staging-security-group graphite-nozzle
-              cf bind-running-security-group graphite-nozzle
-
-              cd graphite-nozzle
-
-              echo "web: graphite-nozzle" > Procfile
-
-              cat <<EOF > manifest.yml
-              ---
-              applications:
+        - task: register-rds-broker
+          config:
+            platform: linux
+            image: docker:///governmentpaas/cf-cli
+            inputs:
+              - name: paas-cf
+              - name: config
+              - name: bosh-CA
+            run:
+              path: sh
+              args:
+                - -e
+                - -c
+                - |
+                  ./paas-cf/concourse/scripts/import_bosh_ca.sh
+                  . ./config/config.sh
+                  echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+                  if cf service-brokers | grep rds; then exit 0;fi
```

Comment:
This check fails, because the broker is called "postgres" not "rds".

I also suggest that you also check the name and the url. something like: `cf service-brokers | grep -e "^postgres[\t ]\+http://${RDS_BROKER_SERVER}"`